nameserver/zone.go

```diff
@@ -18,6 +18,7 @@ var rdnsDomainLen = len(RDNSDomain) + 1
 
 type Zone interface {
 	AddRecord(ident string, name string, ip net.IP) error
+	DeleteRecords(ident *string, ip *net.IP, name *string) (count int)
```

Comment:
It is for consistency with the other methods `Delete*` methods: they return a `LookupError` if no records are deleted...